immut/list/README.md

```diff
@@ -23,7 +23,7 @@ let list = Cons(1, Cons(2, Cons(3, Nil)))
 
 Build a repeated list by using the `repeat()` method:
 ```moonbit
-let list = repeat(3, 1) // of([1, 1, 1])
```

Comment:
This is actually correct since we've removed the sugar `T::[]` in favor of `of([])`

Reply:
We are not using any print or debug statements here. So should the format in the comment match `to_string`, `debug_write` or the actual construction?

As I mentioned in #687, we'd better unify the format.